docs/GettingStarted.md

```diff
@@ -1,15 +1,29 @@
 # Getting Started
 The easiest way to start using Oboe is to build it from source by adding a few steps to an existing Android Studio project.
 
+## Creating an Android app with native support
++ Create a new project: `File > New > New Project`
++ When selecting the project type, select Native C++
++ Finish configuring project
+
 ## Adding Oboe to your project
 
-### 1. Clone the github repository 
-Start by cloning the [latest stable release](https://github.com/google/oboe/releases/) of the Oboe repository, for example: 
+### 1. Clone the github repository
+Start by cloning the [latest stable release](https://github.com/google/oboe/releases/) of the Oboe repository, for example:
 
-    git clone -b 1.1-stable https://github.com/google/oboe
+    git clone -b 1.2-stable https://github.com/google/oboe
 
 **Make a note of the path which you cloned oboe into - you will need it shortly**
 
+If you use git as a VCS, consider adding Oboe as a submodule (underneath your
+app directory)
```

Comment:
Link the word "submodule" to this handy page:
https://gist.github.com/gitaarik/8735255